components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -161,7 +218,11 @@ ext.gleanGenerateMetricsAPI = {
         args "$sourceOutputDir"
         args "-s"
         args "namespace=$fullNamespace"
-        args "$projectDir/metrics.yaml"
+        // Only hardcode the path to the 'metrics.yaml' file if we're not trying to fetch
+        // it from the AAR files.
+        if (!project.ext.has("allowMetricsFromAAR")) {
+            args "$projectDir/metrics.yaml"
+        }
```

Comment:
Is this strictly necessary?  I think `glean_parser` silently ignores missing files.  Just in the interest of simplifying things.

Reply:
Good point. I was torn on this, but we might actually want this behaviour for the GeckoView environment.